File: dotnet/windntrees.core/Application.Core/wwwroot/js/models/Accounting/ChartOfAccount.js
function ChartOfAccount(data) {
    var instance = this;

    instance._datakey = data.Uid;
    instance.Uid = data.Uid;

    instance.Account = ko.observable(data.Account);
    instance.Account.extend({
        required: true,
        maxLength: 20
    });

    instance.Title = ko.observable(data.Title);
    instance.Title.extend({
        required: true,
        maxLength: 50
    });

    instance.Description = ko.observable(data.Description);
    instance.Description.extend({
        maxLength: 255
    });

    instance.AccountTitle = ko.computed(function () {
        return instance.Account() + " - " + instance.Title();
    });

    instance.RowVersion = data.RowVersion;
}

File: dotnet/windntrees.core/DataAccess.Core/Accounting/ChartOfAccount.cs
using Microsoft.AspNetCore.Mvc;
using SharedLibrary.Core.Attributes;
using System.ComponentModel.DataAnnotations;


namespace DataAccess.Core.Poultry
{
    [ModelMetadataType(typeof(ChartOfAccountMetaData))]
    public partial class ChartOfAccount
    {

    }

    public partial class ChartOfAccountMetaData
    {
        [Key]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Accounting.ChartOfAccount), Name = "UID")]
        public System.Guid Uid { get; set; }

        [LocaleMessageRequired]
        [LocaleMessageStringLength(20)]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Accounting.ChartOfAccount), Name = "Account")]
        public string Account { get; set; }

        [LocaleMessageRequired]
        [LocaleMessageStringLength(50)]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Accounting.ChartOfAccount), Name = "Title")]
        public string Title { get; set; }

        [LocaleMessageStringLength(255)]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Accounting.ChartOfAccount), Name = "Description")]
        public string Description { get; set; }

        [Timestamp]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Accounting.ChartOfAccount), Name = "RowVersion")]
        public byte[] RowVersion { get; set; }
    }
}
